src/main/plugins/org.dita.pdf2/resource/messages.xml

```diff
@@ -93,4 +93,8 @@ See the accompanying LICENSE file for applicable license.
     <reason>Found a table row with more entries than allowed.</reason>
     <response/>
   </message>
+  <message id="PDFX013F" type="FATAL">
+    <reason>Generating PDF file '%1' was unsuccessful.</reason>
```

Comment:
I suggest we simplify this to something like:

> The PDF file '%1' could not be generated.

Not sure if we can be more specific w/r/t the reasons for failure, so a rather generic message like this may be best. 

If we _do_ know more about the cause, we might want to add details to the `<response>`.

Reply:
>The PDF file ’%1’ could not be generated.

:+1: sounds much better. I’ll update the PR soon.